tests/unit/src/app/kbv/controllers/prove-identity-another-way.test.js

```diff
@@ -23,48 +20,47 @@ describe("prove identity another way controller", () => {
     expect(controller).toBeInstanceOf(BaseController);
   });
 
-  describe("#locals", () => {
+  describe("#saveValues", () => {
     beforeEach(() => {
-      req.translate = jest.fn();
+      req.session.tokenId = "session-id";
+      req.axios.post = jest.fn();
+      req.form.values.abandonRadio = "continue";
     });
 
-    it("should call super.locals with req and res", () => {
-      const superLocals = jest.spyOn(BaseController.prototype, "locals");
-
-      controller.locals(req, res, next);
+    it("should call abandon endpoint", async () => {
+      req.form.values.abandonRadio = "stop";
```

Comment:
Can we have a test for `abandonRadio` set to something else (should not call abandon endpoint)